test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/LiabilityProcessMoreWeightControllerSpec.scala

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.controllers
+
+import base.unit.ControllerSpec
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.{reset, when}
+import org.mockito.stubbing.OngoingStubbing
+import org.scalatest.Inspectors.forAll
+import org.scalatest.matchers.must.Matchers.convertToAnyMustWrapper
+import play.api.data.Form
+import play.api.http.Status.{BAD_REQUEST, OK, SEE_OTHER}
+import play.api.libs.json.JsObject
+import play.api.mvc.Call
+import play.api.test.Helpers.{redirectLocation, status}
+import play.twirl.api.HtmlFormat
+import uk.gov.hmrc.plasticpackagingtax.registration.connectors.DownstreamServiceError
+import uk.gov.hmrc.plasticpackagingtax.registration.controllers.helpers.LiabilityLinkHelper
+import uk.gov.hmrc.plasticpackagingtax.registration.forms.ProcessMoreWeight
+import uk.gov.hmrc.plasticpackagingtax.registration.models.registration.LiabilityDetails
+import uk.gov.hmrc.plasticpackagingtax.registration.views.html.liability_process_more_weight_page
+import uk.gov.hmrc.play.bootstrap.tools.Stubs.stubMessagesControllerComponents
+
+class LiabilityProcessMoreWeightControllerSpec extends ControllerSpec {
+  private val page                = mock[liability_process_more_weight_page]
+  private val mcc                 = stubMessagesControllerComponents()
+  private val liabilityLinkHelper = mock[LiabilityLinkHelper]
+
+  private val controller =
+    new LiabilityProcessMoreWeightController(authenticate = mockAuthAction,
+                                             mockJourneyAction,
+                                             mockRegistrationConnector,
+                                             mcc = mcc,
+                                             page = page,
+                                             liabilityLinkHelper = liabilityLinkHelper
+    )
+
+  def mockAppConfigIsPreLaunchEnabled(enabled: Boolean): OngoingStubbing[Boolean] =
+    when(config.isPreLaunch)
+      .thenReturn(enabled)
```

Comment:
Is this actually having any effect?  I don't see where the mock AppConfig is being used by the controller.

Reply:
Removed it